site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the resources in a package
+---
+
+<!--mdtogo:Short
+    Print the resources in a package
+-->
+
+{{< asciinema key="pkg-cat" rows="10" preload="1" >}}
+
+Cat prints the resources in a package as yaml to stdout.
+
+Cat is useful for printing only the resources in a package which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt cfg cat [DIR]
+
+DIR:
+  Path to a package directory. Defaults to the current working directory.
```

Comment:
Where's the output of this command described?

Reply:
Updated.